packages/account-v2/src/pages/ConnectedApps/ConnectedApps.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Loader } from '@deriv-com/ui';
+import { useFetchConnectedApps } from '../../hooks';
+import { ConnectedAppsTable } from './ConnectedAppsTable';
+
+export const ConnectedApps = () => {
+    const { data: connectedApps, isLoading } = useFetchConnectedApps();
+
+    return isLoading ? (
+        <div className='flex items-center justify-center h-full'>
```

Comment:
Why is loader component wrapped with a div?

Reply:
Without the div, the loader looks very weird, the div helps keep it in centre along with other styles.